hello-world/AKS/Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM ubuntu:22.04
+FROM mcr.microsoft.com/devcontainers/base:dev-ubuntu-22.04 AS build
+ENV PROJECT_DIR=~/
+WORKDIR ${PROJECT_DIR}/confidential-container-demos
+COPY . ./
+RUN chmod +x hello-world/get_verbose_report.sh && ./hello-world/get_verbose_report.sh
+
+FROM mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
```

Comment:
Do you have a specific one in mind, because searching the whole MCR doesn't seem feasible....

Reply:
Also, for the sidecars specifically, I tried using Azure Linux in Novemeber and couldn't get them to work...

Reply:
> Do you have a specific one in mind, because searching the whole MCR doesn't seem feasible....
Maybe this one: https://mcr.microsoft.com/en-us/product/cbl-mariner/distroless/base/
No idea if it'll work though.